cla_signers_test.py

```diff
@@ -44,24 +73,19 @@ def testValidData(self):
 """
         self.assertEqual(0, self._ValidateData(data))
 
-    def testEmptyStringInvalid(self):
+    def testPeopleSectionEmptyStringInvalid(self):
         data = """\
 people:
   - name:
     email:
     github:
+"""
+        self.assertNotEqual(0, self._ValidateData(data))
```

Comment:
Since there are different error codes I think it would be better to specify how it should fail
```
self.assertEqual(2, self._ValidateData(data))
```

Reply:
In retrospect, the different error codes don't really add much value and I am planning to replace them with a bool (success vs. failure) in the next PR, so this distinction will go away shortly.